scripts/code_coverage_report/generate_code_coverage_report/Sources/CoverageReportGenerator/CoverageReportParser.swift

```diff
@@ -66,7 +66,7 @@ extension CoverageReportRequestData {
       // FirebaseDatabase-ios-FirebaseCore.framework, a regex pattern will be
       // used to exclude results that are not related in terms of the target names.
       let sdk_name = resultBundle.components(separatedBy: "-")[0]
-      let range = NSRange(location: 0, length: sdk_name.utf16.count)
+      let range = NSRange(location: 0, length: target.name.utf16.count)
```

Comment:
This bug is detected [here](https://github.com/firebase/firebase-ios-sdk/pull/7428/checks?check_run_id=1817150397). This is caused by the range is not aligned with the `target.name` in line 75.